app/routes/application.js

```diff
@@ -18,40 +18,15 @@ export default Route.extend(ApplicationRouteMixin, {
     this.get('authManager').initialize();
   },
   model() {
-    return RSVP.hash({
-      notifications: [
-        {
-          title       : 'New Session Proposal for event1 by  user1',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : false
-        },
-        {
-          title       : 'New Session Proposal for event2 by  user2',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : true
-        },
-        {
-          title       : 'New Session Proposal for event3 by  user3',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : false
-        },
-        {
-          title       : 'New Session Proposal for event4 by  user4',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : true
-        },
-        {
-          title       : 'New Session Proposal for event5 by  user5',
-          description : 'Title of proposal',
-          createdAt   : new Date(),
-          isRead      : false
-        }
-      ]
-    });
+    if (this.get('authManager.currentUser')) {
```

Comment:
The correct way of checking if the user is logged in or not is `this.get('session.isAuthenticated')`